content/md/connectivity/manage-event-subscription.md

```diff
@@ -85,3 +89,23 @@ That means that you will receive an event if someone updates the German descript
 ::: info
 If you are not comfortable with Akeneo PIM permissions, check the following article [Set rights on your catalog](access-rights-on-products.html)
 :::
+
+# Debug
+
+Now, everything is set-up, you probably want to be sure your event subscription is working well. 
+
+To do so, we added a button to allow you to download logs related to an event subscription. 
+
+![Download logs button](../img/connection-download-logs-button.png)
+
+In the `.txt` file uploaded on your computer when you click on the `Download logs` button, you have access to several log levels:
+- **Error** logs show you when something went wrong with the connected application. Either it didn't answer quickly enough, or it responded with an error code. 
+- **Warning** logs alert you when your PIM reached the maximum number of events we can send per hour.
+- **Notice** logs tell you why an event hasn't been sent to your connected application. It can be because the connection itself raised the event through the REST API or because the connection has no permission to see the product in question. 
+- **Info** logs inform you when a bunch of events has been sent to your connected application. 
```

Comment:
Do we really need to expand on this ?
It's kinda overwhelming and in my opinion, the messages present in the logs are self explanatory.
Moreover, if we add new logs to the level `warning` for example, the documentation will be outdated and could be confusing.

I also think that the `Want to know more about each type of logs?` is good enough.